app/models/manageiq/providers/ansible_tower/shared/automation_manager/credential.rb

```diff
@@ -0,0 +1,38 @@
+module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Credential
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def create_in_provider(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      credential = manager.with_provider_connection do |connection|
+        connection.api.credentials.create!(params)
+      end
+
+      # Get the record in our database
+      # TODO: This needs to be targeted refresh so it doesn't take too long
+      EmsRefresh.queue_refresh(manager, nil, true)
```

Comment:
So, you're going to change this to depend on https://github.com/ManageIQ/manageiq/pull/14027 ?